gemini-resin-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -29,8 +32,8 @@
   </requiredProperties>
 
   <fileSets>
-    <fileSet filtered="true" packaged="false">
-      <directory>src</directory>
+    <fileSet filtered="true" packaged="true">
+      <directory>src/main/java</directory>
```

Comment:
Wait wat... lol. Does this really work?

Reply:
It does. The idea here is that the generated package would exist in this directory; without this, assuming the package name is "foo", the file structure would be `src/foo/Server.java`